packages/app-store/larkcalendar/api/callback.ts

```diff
@@ -88,7 +89,10 @@ async function getHandler(req: NextApiRequest, res: NextApiResponse) {
       });
     }
 
-    res.redirect(getSafeRedirectUrl(state?.returnTo) ?? "/apps/installed");
+    res.redirect(
+      getSafeRedirectUrl(state?.returnTo) ??
+        getInstalledAppPath({ variant: "calendar", slug: "lark-calendar" })
```

Comment:
This is to support redirecting to the correct category within the Installed Apps page once the app has been installed.